recipes/bioconductor-dexseq/build.sh

```diff
@@ -1,4 +1,16 @@
 #!/bin/bash
 mv DESCRIPTION DESCRIPTION.old
 grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
-$R CMD INSTALL --build .
\ No newline at end of file
+$R CMD INSTALL --build .
+python_scripts=$PREFIX/lib/R/library/DEXSeq/python_scripts
+sed -i.bak '1s|^|#!/usr/bin/env python\'$'\n|g' $python_scripts/dexseq_count.py
+sed -i.bak '1s|^|#!/usr/bin/env python\'$'\n|g' $python_scripts/dexseq_prepare_annotation.py
+if [ "$PY3K" == 1 ]; then
```

Comment:
I tried that, and added `set -x` to the build script. Seems that `$PY3K` is set to `0`, and thus the `2to3` commands are skipped. I'm going to try adding `python` to the `host` section of the `meta.yaml` file to see if that sets `$PY3K`.

Reply:
Well, that didn't work either. `$PY3K` remains set to `0`. Perhaps that is broken or not the correct way to detect that we're building against Python 3? [The documentation regarding build variables](https://conda.io/docs/user-guide/tasks/build-packages/environment-variables.html) is a bit vague:
> PY3K | 1 when Python 3 is installed in the build prefix, otherwise 0.

Perhaps using `$PY_VER` is more appropriate?

Reply:
Interesting... `$PY_VER` is clearly set to `2.7`... Looking at the build output, it seems that python 2.7 is used during the build, but that python 3.6 is used when installed and tested. Is there a way to force these to be the same version?